jwql/instrument_monitors/miri_monitors/cosmic_ray_monitor.py

```diff
@@ -0,0 +1,628 @@
+#! /usr/bin/env python
```

Comment:
@mengesser It looks like this file is riddled with merge conflicts.  Could you solve them before I give this PR a proper review?

Reply:
Fixed